bintrace.c

```diff
@@ -25,10 +25,20 @@
 
 #define MAX_HIT_COUNT 0xffffffffffffffff
 
+#define COL_W 25
+
 struct hist_entry_t
 {
 	uint8_t break_index;
 	size_t hit_count;
+	struct user_regs_struct regcaps[5];
+};
+
+struct flags
```

Comment:
Let's rename this to `struct program_opts`. Name `flags` might be confused with something different, I think.